test/end-to-end/auth.js

```diff
@@ -1,27 +1,33 @@
 const { ErrorTypes } = require("../../src/api/middleware/errorHandler");
 const Account = require("../../src/models/Account");
 
+const test_god_token  = "testing_is_cool73";
 describe("Register endpoint test", () => {
     describe("Input Validation (unsuccessful registration)", () => {
-        describe("username", () => {
+        describe("email", () => {
             test("should be required", async () => {
                 const res = await request()
                     .post("/auth/register")
-                    .send({});
+                    .send({
+                        god_token: test_god_token,
+                    });
 
                 expect(res.status).toBe(422);
                 expect(res.body).toHaveProperty("error_code", ErrorTypes.VALIDATION_ERROR);
                 expect(res.body).toHaveProperty("errors");
                 expect(res.body.errors).toContainEqual({
                     "location": "body",
-                    "msg": "Username is required",
-                    "param": "username",
+                    "msg": "Email is required",
+                    "param": "email",
```

Comment:
This (and all other tests that can do so) should be using the abstractions for testing validators that were introduced in the previous PR.